packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -297,20 +293,11 @@ EmbedFlutterFrameworks() {
   # Embed the actual Flutter.framework that the Flutter app expects to run against,
   # which could be a local build or an arch/type specific build.
 
-  # Prefer the hidden .ios folder, but fallback to a visible ios folder if .ios
```

Comment:
It's okay to use the `BUILT_PRODUCTS_DIR` engine for add-to-app because assemble put it in the right place.